src/Particle.js

```diff
@@ -609,29 +616,29 @@ class Particle {
 	 * @param  {String} [options.org]     Organization Slug
 	 * @param  {String} [options.product] Events for this product ID or slug
 	 * @param  {String} options.auth     Access Token
-	 * @return {Promise} If the promise resolves, the resolution value will be an EventStream object that will
+	 * @returns {Promise} If the promise resolves, the resolution value will be an EventStream object that will
 	 * emit 'event' events.
 	 */
-	getEventStream({ deviceId, name, org, product, auth, context }) {
```

Comment:
another unused `context` instance - this one less controversial since i don't think it's relevant here?